### PR TITLE
Xorg server 1.20 update

### PR DESCRIPTION
**Pull request by lrusak**
This is a massive update to xorg-server 1.20

This cannot be merged until nvidia updates their drivers to be compatible

### CODE REVIEW

**Comment by ksooo** [comment]
> This cannot be merged until nvidia updates their drivers to be compatible

 They finally delivered. ;-) 

http://www.nvidia.com/Download/driverResults.aspx/134859/en-us
http://www.nvidia.com/Download/driverResults.aspx/135161/en-us

File: packages/x11/xserver/xorg-server/package.mk [hunk 1]
```diff
@@ -17,13 +17,13 @@
 ################################################################################
 
 PKG_NAME="xorg-server"
-PKG_VERSION="1.19.6"
-PKG_SHA256="a732502f1db000cf36a376cd0c010ffdbf32ecdd7f1fa08ba7f5bdf9601cc197"
+PKG_VERSION="1.19.99.904"
```
Comment (CvH) [inline packages/x11/xserver/xorg-server/package.mk]: xorg-server-1.20.0 is out :)

File: packages/x11/driver/xf86-video-vmware/patches/xf86-video-vmware-01-saa-build-compatability-with-xorg-server-1.20.patch [hunk 2]
```diff
@@ -0,0 +1,36 @@
+From patchwork Tue Nov  7 20:45:11 2017
+Content-Type: text/plain; charset="utf-8"
+MIME-Version: 1.0
+Content-Transfer-Encoding: 7bit
+Subject: [xf86-video-vmware] saa: Build compatibility with xserver 1.20
```
Comment (CvH) [inline packages/x11/driver/xf86-video-vmware/patches/xf86-video-vmware-01-saa-build-compatability-with-xorg-server-1.20.patch]: xf86-video-vmware-13.3.0 is out where 1.20 compatibility is included

**Comment by MilhouseVH** [comment]
@lrusak: http://ix.io/1cN7

I've left it to you to squash/rebase (after mesa?) as appropriate.

**Comment by lrusak** [comment]
updated

**Pull request merged**